packages/polyserve/src/compile-middleware.ts

```diff
@@ -72,6 +74,9 @@ export function babelCompile(
       if (isPolyfill.test(request.url)) {
         return false;
       }
+      if (shouldNotCompile.test(request.url)) {
```

Comment:
Maybe use `('nocompile' in request.query)` or similar so that `nocompile` doesn't have to be the first query parameter (e.g. `foo&nocompile`).

Reply:
https://expressjs.com/en/4x/api.html#req.query

Reply:
Ah, TIL. Done :+1: